### ISSUE TITLE
NVDA not announcing table cell position in UIA implementation

### ISSUE THREAD

**Issue by danghoaiphuc** [nvaccess/nvda#19145]
We are developing UIA support for the TRichView editor component used in the Sao Mai Braille translation software. In our implementation:
- The Table implements IGridProvider and ITableProvider.
- The Cell in the table implements IGridItemProvider and ITableItemProvider.

When we create a table in the document and move the cursor from outside the table into a cell, the Speech Viewer displays:
"table with 2 rows and 2 columns
data item row 1 column 1
document Calibri 10.0 pt Today is Monday"

This shows that our implementation of IGridProvider, ITableProvider for the table and IGridItemProvider, ITableItemProvider for the cell is working correctly.
However, NVDA actually reads out:
"table with 2 rows and 2 columns
document Calibri 10.0 pt two"

Could you please let me know why the Speech Viewer shows the correct cell position (row 1, column 1), but NVDA does not announce it when reading aloud?
Many thanks and look forward to your support.

**Comment by SaschaCowley** [comment]
Hi @danghoaiphuc, are you able to provide us a with a minimal code sample, or an executable that exhibits this behaviour so we can investigate?

**Comment by Emil-18** [comment]
@danghoaiphuc Could you set NVDA's logging level to debug, navigate the table, press NVDA+f1 to bring up the log viewer, save the log as a file, and provide it here?

**Comment by seanbudd** [comment]
Closing as we haven't received required information
